src/main/java/domen/rideapp/infrastructure/repository/InMemoryPricingConfigRepository.java

```diff
@@ -0,0 +1,16 @@
+package domen.rideapp.infrastructure.repository;
+
+import domen.rideapp.domain.model.PricingConfig;
+import domen.rideapp.domain.repository.PricingRepository;
+
+import java.util.Optional;
+
+public class InMemoryPricingConfigRepository implements PricingRepository {
+    @Override
+    public Optional<PricingConfig> getDefaultConfig() {
```

Comment:
Uważam że PricingConfig to idealny przypadek użycia wartości z application.properties - bardzo łatwo można je zmieniać po starcie aplikacji, zawsze będzie od razu dostępny (czyli nie musimy się bać optionala). Wówczas zrezygnowałbym z PricingRepository. Co innego gdy chcesz dynamiczną walutę zmieniającą się w trakcie działania aplikacji, to wtedy te podejście miałoby większy sens, ale póki co jest ona stała/tworzona przy starcie aplikacji